UnityProject/Assets/Scripts/Player/CrossedBehaviors/Slippery.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Slippery : MonoBehaviour
+{
+	private RegisterItem registerItem;
+
+	private void Awake()
+	{
+		registerItem = GetComponent<RegisterItem>();
+	}
+
+	private void OnEnable()
+	{
+		registerItem.OnCrossed.AddListener(Slip);
+	}
+
+	private void OnDisable()
+	{
+		registerItem.OnCrossed.RemoveListener(Slip);
+	}
+
+	private void Slip()
+	{
+		if (MatrixManager.IsSpaceAt(transform.position.CutToInt()))
```

Comment:
don't use transform.position for any serverside calculations (and preferably clientside, too)
you've got RegisterTile reference, use it instead
```suggestion
		if (MatrixManager.IsSpaceAt(registerItem.WorldPosition))
```